cloudbuild-release.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2019 The Forseti Security Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This defines the build steps of Forseti docker images. The google container
```

Comment:
Can we add some comments to describe what this file specifically is for?  So that I can diferentiate the `-dev.yaml`, `-release.yaml`, and just the `cloudbuild.yaml`?